working-open.md

```diff
@@ -0,0 +1,46 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
```

Comment:
What does the phrase "new economy" refer to? It was historically used in relation to [internet related innovation](https://www.nber.org/papers/w7833), however, I think you might be referring to [something like this](https://neweconomy.net/about/what-is-the-new-economy)?

Reply:
Good point. I'd be satisfied with a link out to something, rather than diving into the meaning in this doc itself, if that works for others

Reply:
Or we could start a glossary in the handbook, and start corralling this stuff :)

Reply:
Yeah I was borrowing "new economy" as shorthand for the three aspects they mention: " just, sustainable, and democratic future" + solidarity economy etc... I think we can use a non-jargon word here (and maybe crib from our mission/vision), but think a glossary of things would be good? I'm filing an issue of glossary terms here --> #4

Reply:
Okay added suggested language below, I'm resolving this for now